Sources/CXExtensions/DelayedAutoCancellable.swift

```diff
@@ -7,26 +7,28 @@ import CXFoundation
 extension Cancellable {
     
     public func cancel<S: Scheduler>(after interval: S.SchedulerTimeType.Stride, tolerance: S.SchedulerTimeType.Stride = .zero, scheduler: S, options: S.SchedulerOptions? = nil) -> DelayedAutoCancellable {
-        return DelayedAutoCancellable(cancel: self, after: interval, tolerance: tolerance, scheduler: scheduler, options: options)
+        return DelayedAutoCancellable(cancel: self.cancel, after: interval, tolerance: tolerance, scheduler: scheduler, options: options)
     }
 }
 
 public final class DelayedAutoCancellable: Cancellable {
     
-    private let canceller: Cancellable
+    private var cancelBody: (() -> Void)?
```

Comment:
Why is canceller a closure? Two `Cancellable` make it easy to know what we are doing here.

Reply:
Store a protocol type could result heap allocation due to existential container.

Reply:
As for scheduleCanceller, the scheduler gives us an existential container anyway, so I just take it.

Reply:
🤣, ok, ok.